test/e2e/performanceprofile/functests/1_performance/performance.go

```diff
@@ -1018,6 +1019,52 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			})
 		})
 	})
+
+	Context("Offlined CPU API", func() {
+		It("should set offline cpus after deploy PAO", func() {
+			By("Backing up the profile")
+			initialProfile := profile.DeepCopy()
+
+			// Create new performance with offlined
+			reserved := performancev2.CPUSet("0")
+			isolated := performancev2.CPUSet("1")
+			offlined := performancev2.CPUSet("2-3")
```

Comment:
we assume the node has >= 4 cpus, ideally we should check that's the case, perhaps skipping if the node has less than X cpus - to make sure we offline only a tiny fraction of them